docs/azure_arc_jumpstart/azure_arc_servers/azure_stack_hci/azure_stack_hci_windows/_index.md

```diff
@@ -0,0 +1,226 @@
+
+    To be able to complete the scenario and its related automation, an Azure service principal assigned with the “Contributor” role is required. To create it, login to your Azure account using PowerShell and run the below command.
+
+    ```powershell
+    Connect-AzAccount
+    $sp = New-AzADServicePrincipal -DisplayName "<Unique SP Name>" -Role 'Contributor'
+    ```
+
+    For example:
+
+    ```powershell
+    $sp = New-AzADServicePrincipal -DisplayName "AzureStackHCI-VM-Jumpstart" -Role 'Contributor'
+    ```
+
+    This command will create a variable with a secure string as shown below:
+
+    ```shell
+    Secret                : System.Security.SecureString
+    ServicePrincipalNames : {XXXXXXXXXXXXXXXXXXXXXXXXXXXX, http://AzureStackHCI-VM-Jumpstart}
+    ApplicationId         : XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    ObjectType            : ServicePrincipal
+    DisplayName           : AzureStackHCI-VM-Jumpstart
+    Id                    : XXXXXXXXXXXXXXXXXXXXXXXXXXXX
+    Type                  :
+    ```
+
+    To expose the generated password use this code to export the secret:
+
+    ```powershell
+    $BSTR = [System.Runtime.InteropServices.Marshal]::SecureStringToBSTR($sp.Secret)
+    $UnsecureSecret = [System.Runtime.InteropServices.Marshal]::PtrToStringAuto($BSTR)
+    ```
+
+    Copy and save the Service Principal ApplicationId and Secret as you will need it for later on in the automation.
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/powershell/module/az.resources/new-azadserviceprincipal?view=azps-5.4.0)**
+
+* Enable CredSSP in the Host Server
+
+    The Powershell script leverages [CredSSP](https://docs.microsoft.com/en-us/windows/win32/secauthn/credential-security-support-provider) to delegate credentials from the management server to the target server for remote authentication. It's required to allow the host server to receive credentials from a remote computer.
+
+    ```powershell
+    Enable-WSManCredSSP -Role Server
+    ```
+
+    ![Screenshot Enable-WSManCredSSP Output](./01.png)
+
+    Confirm if CredSSP enabled the host server to receive credentials:
+
+    ```powershell
+    Get-WSManCredSSP
+    ```
+
+    ![Screenshot Enable-WSManCredSSP Output](./02.png)
+
+* Create a Virtual Switch
+
+    > **Note: If you already have a Virtual Switch with Internet access you can skip this step.**
+
+    It's required to have a [Virtual Switch](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/create-a-virtual-switch-for-hyper-v-virtual-machines#create-a-virtual-switch-by-using-windows-powershell) created in advance, before running the Powershell script. If not yet created, you can use the following powershell command in your host server:
+
+    ```powershell
+    New-VMSwitch -Name ExternalSwitch  -NetAdapterName Internet-Access -AllowManagementOS $true
+    ```
```

Comment:
Any specific reason for not having both CredSSP and vSwitch automation as part of the PS script?

Reply:
The vSwitch creation can be added to the PS script. The CredSSP I don't think so. I'm enabling it in the management computer in the PS script. However, in the host server it needs to be manually done, since I can't do a remote PS session without it.